.github/workflows/conda-python-tests.yml

```diff
@@ -31,4 +30,3 @@ jobs:
       run: |
         conda install pytest
```

Comment:
maybe what might fix this is instead of running "conda install pytest" here we just run "pip install pytest"?

Reply:
There are few things we need to make consistent. One, we are running tests on conda but publishing pip package and not a conda package. If that is the case, then we should validate the package using python based environments. Or maybe have separate tests for pip based and conda based tests. That's why I created these two pipelines.

Reply:
discussed in teams as well:
Our package will be the same since it's only installed via pip.
Some other packages like pytorch will be much faster when installed via conda than pip, so to make our tests run faster it makes more sense to install via conda.
Also conda gives us a lot more flexibility in terms of the packages we can install, and whether we want to use multiple envs in CI (I haven't needed to do that yet though).
The main reason to use conda is just our tests will run faster with some frameworks like pytorch.

Reply:
If we add tests in conda and regular python, then it kind of defeats the purpose of adding conda, since it was just meant to make the tests run faster?  What do you think?